src/AlixBa/Addic7edSubtitles/Helpers/Episode.php

```diff
@@ -51,8 +51,8 @@ public function __construct($episodeFilename)
     {
         preg_match($this->pattern, $episodeFilename, $matches);
 
-        $this->showName          = $matches['showname'];
-        $this->sanitizedShowName = self::sanitizeShowName($matches['showname']);
+        $this->showName          = self::capitalizeShowName($matches['showname']);

```

Comment:
Actually I just realized that the property "showName" is never used. 

We could entirely capitalize it (not only the first letters) and modify the ShowsUpdater to also capitalize shows' name. Unless this could cause troubles (same show name but different case?). It could lead to a better show's name matching.

Another thing could be to do the capitalizing inside the sanitize method and don't touch the showName. We could display the show name in the output `printf("Missing show [%s].\n", $episode->sanitizedShowName);`